src/lib/style/bootstrap4_backports/_base-import.scss

```diff
@@ -158,4 +159,49 @@
       $active-background: SRC.$gray-light
     );
   }
+
+  // Primary button based on Synapse Design System
+  // See https://www.figma.com/file/0oPm5lLSUva8kyfVNMS6FA/Synapse-Style-%26-Component-Library?node-id=53%3A1189
+  .btn-sds-primary {
+    @include button-variant(
+      $background: map.get(SRC.$primary-color-palette, 500),
+      $border: map.get(SRC.$primary-color-palette, 500),
+      $hover-background: map.get(SRC.$primary-color-palette, 600),
+      $hover-border: map.get(SRC.$primary-color-palette, 600),
+      $active-background: map.get(SRC.$primary-color-palette, 700),
+      $active-border: map.get(SRC.$primary-color-palette, 700)
+    );
+    border-radius: 0px;
+    font-weight: 900;
+    &:hover {
+      box-shadow: none; // ?
```

Comment:
Need to clarify w/ design about this but IMO a box shadow on these new buttons would look out-of-place.